modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -46,6 +46,8 @@ public class UpgradeCatchAllCheck {
 		CookieKeys.COMMERCE_CONTINUE_AS_GUEST;
 		CookieKeys.ID;
 		CookieKeys.SCREEN_NAME;
+
+		ServiceTrackerList<Class<T>, Class<T>> serviceTrackerList = ServiceTrackerListFactory;
```

Comment:
@TamyrisTorres, can we change the second `Class<T>` to `Class<E>`, for example?

My idea is make clearer in the test that the check is intentionally keeping the first type parameter instead of the second one, at this moment, it isn't obvious since there are 2 `Class<T>` and the check keep only 1 of them, but which one? Does it make sense to you?

Reply:
resolved!